apps/communications/contacts/test/unit/views/form/boot_test.js

```diff
@@ -0,0 +1,175 @@
+/* global LazyLoader, Form, FormUI */
+
+'use strict';
+
+require('/shared/test/unit/load_body_html_helper.js');
+require('/shared/js/lazy_loader.js');
+requireApp('communications/contacts/views/form/js/boot.js');
+
+
+suite('Boot', function() {
+
+  setup(function() {
+    loadBodyHTML('/contacts/views/form/form.html');
+  });
+
+  teardown(function() {
+    document.body.innerHTML = '';
+  });
+
+  suite('DOMContentLoaded', function() {
+    var stub;
+    setup(function() {
+      stub = this.sinon.stub(LazyLoader, 'load', function(files, callback) {
+        callback();
+      });
+      window.dispatchEvent(new CustomEvent('DOMContentLoaded'));
+    });
+
+    teardown(function() {
+      LazyLoader.load.restore();
+    });
+
+    test(' > LazyLoader must be called 2 times', function() {
+      // We must have 2 calls to LazyLoader
+      assert.isTrue(stub.calledTwice);
+    });
+
+    test(' > First call must ensure localization', function() {
+      // First of all we need to bring l10n library for localization
+      var firstCall = stub.getCall(0);
+      assert.isTrue(Array.isArray(firstCall.args[0]));
+      assert.equal(firstCall.args[0][0], '/shared/js/l10n.js');
+    });
+
+    test(' > Second call must load the template', function() {
+      // Secondly we lazy load the panel, so the param must be the element
+      // of the DOM to fill with the template
+      var secondCall = stub.getCall(1);
+      assert.equal(secondCall.args[0][0].id, 'view-contact-form');
+    });
+  });
+
+  suite('Loaded', function() {
+    var realFormController, realFormUI, realMozSetMessage;
+    suiteSetup(function() {
+      realFormController = window.Form;
+      realFormUI = window.FormUI;
+
+      window.Form = {
+        init: function foo() {}
+      };
+
+      window.FormUI = {
+        init: function foo() {}
+      };
+
+      realMozSetMessage = navigator.mozSetMessageHandler;
+      navigator.mozSetMessageHandler = function foo() {};
+    });
+
+    suiteTeardown(function() {
+      window.Form = realFormController;
+      window.FormUI = realFormUI;
+      navigator.mozSetMessageHandler = realMozSetMessage;
+
+      realFormController = null;
+      realFormUI = null;
+      realMozSetMessage = null;
+    });
+
+    setup(function() {
+      this.sinon.stub(LazyLoader, 'load', function(files, callback) {
```

Comment:
Should we move it to the `setup` phase of the wrapping suite, `Boot`, in this case? I see you are using it again:
https://github.com/mozilla-b2g/gaia/pull/30694/files#diff-ced686944c86edddfaafc7ed677137fbR134